service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -65,6 +65,11 @@ public boolean isRestrictedFromCaseSubmission(String localAuthorityName) {
             createLDUser(Map.of("localAuthorityName", LDValue.of(localAuthorityName))), false);
     }
 
+    public boolean isSendToSolicitorEnabled(String localAuthorityName) {
```

Comment:
it would be nice to improve method name and maybe toggle name. Now it does not tell what it is about, now reference that it is about email/notification

Reply:
updated method name + LD flag